game/resource/English/ability/items/tooltip_ancient_janggo.txt

```diff
@@ -4,7 +4,7 @@
 
 "DOTA_Tooltip_Ability_item_ancient_janggo_2"            "Drum of Endurance"
 "DOTA_Tooltip_Ability_item_recipe_ancient_janggo_2"     "Drum of Endurance Recipe"
-"DOTA_Tooltip_Ability_item_ancient_janggo_2_Description"                        "<h1>Active: Endurance</h1> Gives +%bonus_attack_speed_pct% attack speed and +%bonus_movement_speed_pct%%% movement speed to nearby allies for %duration% seconds. <br><br>Radius: %radius%\n<h1>Passive: Swiftness Aura</h1>Gives %bonus_aura_movement_speed% movement speed to nearby allies. <br><br>Radius: %radius%"
```

Comment:
drums is missing the change in the t1 item, arguagbly not needed anywhere but t1